__test__/creds/fixtures/CredentialEmailInvalid.json

```diff
@@ -3,14 +3,14 @@
   "id": "81eed612-0bce-4eb5-a88f-d96c34cc5431",
   "issuer": "did:ethr:0x1a88a35421a4a0d3e13fe4e8ebcf18e9a249dc5a",
   "issuanceDate": "2019-08-15T21:18:25.058Z",
-  "identifier": "credential-cvc:Email-v1",
-  "expirationDate": null,
+  "identifier": "credential-cvc:Email-v3",
```

Comment:
Is there are reason you jump from v1 to v3? Does v2 exist for these credentials?